frontend/src/services/configMapService.ts

```diff
@@ -1,33 +1,33 @@
-import axios from 'axios';
+import {
+  k8sCreateResource,
+  k8sDeleteResource,
+  k8sGetResource,
+  k8sUpdateResource,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import { ConfigMapModel } from '../models';
 import { ConfigMap, DeleteStatus } from '../types';
 
 export const getConfigMap = (projectName: string, configMapName: string): Promise<ConfigMap> => {
-  const url = `/api/configmaps/${projectName}/${configMapName}`;
-  return axios.get(url).then((response) => {
-    return response.data;
+  return k8sGetResource<ConfigMap>({
```

Comment:
Maybe we can get rid of the braces and `return` if we are not going to do any other operation before the return statement?

Reply:
I think if we want this kinda style, we should just force it in the linter. This is a preference thing of code style and we could probably disagree until we go blue in the face lol. Let us talk about linter improvements. 

I logged https://github.com/opendatahub-io/odh-dashboard/issues/469 to have that conversation.